shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -75,6 +75,8 @@ const ADVANCED = 'advanced';
 const HARVESTER = 'harvester';
 const HARVESTER_CLOUD_PROVIDER = 'harvester-cloud-provider';
 
+const noneOption = { label: '(None)', value: '' };
```

Comment:
The label here should probably be localized

Reply:
I restored the default label for PSA and PSP, also added i18n for `profileOptions`.